js/annyang.js

```diff
@@ -136,7 +136,7 @@
       recognition.interimResults = true;
 
       // Sets the language to the default 'en-US'. This can be changed with annyang.setLanguage()
-      recognition.lang = 'en-US';
+      recognition.lang = config.locale;
```

Comment:
This should be set in the Annyang Service. Not in the Annyang library (which may be updated one day).